### DISCUSSION TITLE
'Definition' list

### DISCUSSION THREAD

**Discussion by bjarkih1977** [jendrikseipp/rednotebook#601] [Ideas]
Not sure if the title is clear.  English is not my native language.  It would be nice to be able to create a personal database (glossary) for often used phrases or words that the user can link to for future reference.   Then the user could link to that in entries later on.

**Comment by jendrikseipp** [comment]
You could add this definitions list to a specific day and then link to that day from other days. See the help document (Ctrl+H) for how to link to other days.

**Reply by bjarkih1977** [reply]
Not very easy to use if you forget the date.
